src/content/en/fundamentals/engage-and-retain/push-notifications/handling-messages.markdown

```diff
@@ -0,0 +1,448 @@
+---  
+layout: shared/narrow
+title: "Handling messages"
+description: ""
+published_on: 2016-07-01
+updated_on: 2016-07-01
+order: 40
+translation_priority: 0
+authors:
+  - josephmedley
+---
+
+<p class="intro"> Way back at the [beginning of this section](index#anatomy), we
```

Comment:
Markdown isn't being rendered, either change to HTML for the link or add markdown=1